collections/brandedUserProfile.js

```diff
@@ -232,7 +232,8 @@ BrandedUserSchema = new SimpleSchema({
     permittedUserMutable: fields.bool(o),
     twilioCode: fields.number(o),
     twilioLookUpValid: fields.bool(o),
-    wrongNumber: fields.bool(o)
+    wrongNumber: fields.bool(o),
+    permanentlySoftStopped: {type: Boolean, optional: true, defaultValue: false}
```

Comment:
Sorry I didn't actually look at the change earlier and avoid a round trip, but is there any reason not to use the below suggestion?  I believe it's the same thing, and is consistent with the rest of the schema.
```suggestion
    permanentlySoftStopped: fields.bool(o)
```

Reply:
it doesn't include the defaultValue

Reply:
Oh, right; I misread that as saying that there is no defaultValue (i.e., the statement that a defaultValue exists is false), rather than saying that there is a defaultValue, and that value is false.